.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+## Changes
+<!-- Summary of your changes that are easy to understand -->
+
+## Tests
+<!-- How is this tested? -->
```

Comment:
Add a checklist:

- [ ] `make test` run locally
- [ ] relevant change in `docs/` folder
- [ ] covered with integration tests in `internal/acceptance`
- [ ] relevant acceptance tests are passing
- [ ] using Go SDK